kernel/univ.mli

```diff
@@ -349,6 +349,14 @@ sig
 
 end
 
+type 'a univ_abstracted = {
+  univ_abstracted : 'a;
+  nbound_univs : int;
+}
+(** A value with [nbound_univs] bound universe levels. *)
```

Comment:
Changed to use just an `int` as we have no use for more, if that changes we can put whatever we need in the type later.